go.mod

```diff
@@ -1,8 +1,16 @@
 module github.com/container-storage-interface/spec
 
-go 1.16
+go 1.19
```

Comment:
is this version actually required to build CSI (w/ reasonable deps)? if not, why not bump to 1.17 or 1.18 instead? CSI module doesn't need to be bleeding edge (and thereby force downstream users to use the latest Go)

Reply:
sure @jdef , I have declared it on 1.18 in the last refresh.